src/acvp.h

```diff
@@ -167,7 +168,7 @@ typedef enum acvp_prereq_mode_t {
 } ACVP_PREREQ_ALG;
 
 #define ACVP_KDF135_SNMP_ENGID_MAX 32
-#define ACVP_KDF135_SNMP_SKEY_MAX 32
```

Comment:
I would like to keep these separate just in case the strength of one iKDF s changed and the other is not.
Otherwise everything else looks good !